entities/entities.go

```diff
@@ -468,6 +469,22 @@ func getObjectStatus(obj *unstructured.Unstructured, gvrk kuber.GroupVersionReso
 	}
 }
 
+func getObjectMeta(u *unstructured.Unstructured) (map[string]interface{}, error) {
+	meta, found, err := unstructured.NestedFieldNoCopy(u.Object, "metadata")
+	if !found || err != nil {
+		return nil, fmt.Errorf("unable to obtain metadata, error: %w", err)
+	}
+	metadataMap, ok := meta.(map[string]interface{})
+	if !ok {
+		return nil, fmt.Errorf("metadata is type %T, map[string]interface{} expected", meta)
+	}
+	return map[string]interface{}{
+		"namespace":       metadataMap["namespace"],
+		"name":            metadataMap["name"],
+		"ownerReferences": metadataMap["ownerReferences"],
```

Comment:
We don't really need the owner references field. From what I understand from the entities-definitions code, we can replace it with a flag.